confiture-web-app/src/pages/audit/AuditOverviewPage.vue

```diff
@@ -94,25 +98,47 @@ function focusPageHeading() {
       </button>
     </div>
 
+    <BackLink
+      v-if="isConnected"
+      label="Retourner à mes audits"
+      :to="{ name: 'account-dashboard' }"
+    />
+
     <div class="content">
-      <h1 class="fr-mb-6w">{{ audit.procedureName }}</h1>
+      <template v-if="isConnected">
```

Comment:
Changer la condition de `is connected` à `is connected and owns this audit`

Reply:
Sinon lorsqu'on est connecté, et qu'on va sur un audit pas associé avec son compte, on ne peut pas accéder à l'audit

Reply:
J'ai mis à jour la condition dans le composable @hissalht.

Reply:
Ah ouais t'as directement changé le code de `isConnected`. C'est fine mais faudrait renommer `isConnected` je pense. Quand je lis `isConnected` je m'attends pas à ce qu'il y ait autre chose de checké que "est-ce que l'utilisateur est connecté"

D'ailleurs, est-ce que ça serait pas plutôt "isLoggedIn" ou "isSignedIn" ?

Reply:
Oui effectivement le composable check juste si on est connecté. Et j'ai bougé l'autre condition aux 2 endroits où je m'en servais.